test/clojure/nrepl/util/completion_test.clj

```diff
@@ -45,6 +45,7 @@
            (candidates "java.lang.System/out")))
 
     (is (some #{"String/valueOf"} (candidates "String/")))
+    (is (every? #(= 1 %) (vals (frequencies (candidates "String/")))))
```

Comment:
Might be a good idea to add some helper named `unique-candidates?` and use it in the other completion tests as well.

Reply:
I can do that. I realized that I could just do `(apply distinct? (candidates "String/"))` instead, though.